openstf/pipeline/train_model_sklearn.py

```diff
@@ -56,8 +42,7 @@ def train_model_pipeline(pj: dict, check_old_model_age: bool = True,
 
     # Add features
     data_with_features = TrainFeatureApplicator(TRAIN_HORIZONS,
-                                                features=db.get_featureset(
-                                                    hyper_params["featureset_name"])
+                                                features=pj['features_set']
```

Comment:
I personally think using just the regular 4 spaces for indentation makes things easier to read. But I'm not sure if this is a convention already. This is what I mean:
```python
data_with_features = TrainFeatureApplicator(
    TRAIN_HORIZONS,
    features=db.get_featureset(hyper_params["featureset_name"]),
    features=pj['features_set']
 ).add_features(validated_data)
```

Also, it looks like a `,` is missing after `features=db.get_featureset(hyper_params["featureset_name"])`

Reply:
I see there where some more issues with this statement, but you already solved them :)